storage/storage.c

```diff
@@ -1149,9 +1179,11 @@ static secbool unlock(const uint8_t *pin, size_t pin_len,
   // Now we can check for wipe code.
   ensure_not_wipe_code(unlock_pin, unlock_pin_len);
 
-  // Get the pin failure counter
+  // Get the pin failure counter. To ensure correct exponential backoff,
+  // synchronize counters in case they diverged. However, we are unaware of any
+  // way how the Optiga counter could be ahead of the MCU counter.
```

Comment:
i thought the obvious answer is "user manages to unplug device at exactly the right moment"?
(more like, the MCU counter shouldn't be ahead of Optiga unless Optiga is bad)